concordium-sdk/pom.xml

```diff
@@ -4,9 +4,14 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
+    <parent>
+        <groupId>com.concordium.sdk</groupId>
+        <artifactId>concordium-sdk-base</artifactId>
+        <version>6.0.1-SNAPSHOT</version>
+    </parent>
+
     <groupId>com.concordium.sdk</groupId>
     <artifactId>concordium-sdk</artifactId>
-    <version>6.0.1-SNAPSHOT</version>
```

Comment:
Does this change anything about the release process? https://github.com/Concordium/concordium-java-sdk/blob/main/release/README.md

E.g. does one now have to run `mvn versions:set -DnewVersion=$VERSION` from the root of the repository? As opposed to the root of the java sdk. ?

How does this affect the release process outlined in `release.yml` ?

Reply:
Yes, it needs to be run from the root. :disappointed:
I've updated the release/README to reflect that.

I've updated the release.yml comments to note that we also get an AAR file in the process, but I don't think it affects it otherwise. :thinking: